### ISSUE TITLE
Enchantment Window Empty

### ISSUE THREAD

**Issue by Merancapeman** [JackFred2/JacksServerSideTweaks#1]
As of the latest version for Fabric, there are no enchantment options.

**Comment by JackFred2** [comment]
If you're playing with modded enchantments I think I know the issue; haven't gotten around to implementing a paginated selector window yet since vanilla fits on one page. I'll fix this tomorrow afternoon

**Comment by Merancapeman** [comment]
Thank you! If modded enchantments are causing the issue though I'd rather have your mod because the modded enchantments I've installed don't seem to work anyway. Frankly I haven't seen any modded enchants work for server side fabric mods. Your work rocks, much appreciated!

**Comment by JackFred2** [comment]
Thanks for the kind comments!It's not that other mods are breaking it, it's just something that should be implemented but just wasn't yet. Once I've pushed the update they should all show.On 15 Apr 2023 04:01, Merancapeman ***@***.***> wrote:
Thank you! If modded enchantments are causing the issue though I'd rather have your mod because the modded enchantments I've installed don't seem to work anyway. Frankly I haven't seen any modded enchants work for server side fabric mods. Your work rocks, much appreciated!
